esp/esp/program/models/__init__.py

```diff
@@ -1925,6 +1928,7 @@ def install():
     print "Installing esp.program initial data..."
     install_class()
 
-# The following are only so that we can refer to them in caching Program.getModules.
+# The following are only so that we can refer to them in caching
 from esp.program.modules.base import ProgramModuleObj
 from esp.program.modules.module_ext import ClassRegModuleInfo, StudentClassRegModuleInfo
+from esp.resources.models import ResourceType
```

Comment:
I'm assuming you tried adding the import at the top and got circular import errors?